addons/html_editor/static/src/editor/core/delete_plugin.js

```diff
@@ -397,13 +516,9 @@ export class DeletePlugin extends Plugin {
             const previous = previousLeaf(endLeaf, this.editable, true);
             if (previous && closestElement(previous).isContentEditable) {
                 [endContainer, endOffset] = [previous, nodeSize(previous)];
-                commonAncestorContainer = getCommonAncestor(
-                    [startContainer, endContainer],
-                    this.editable
-                );
             }
         }
-        return { ...selection, endContainer, endOffset, commonAncestorContainer };
+        return { ...range, endContainer, endOffset };
     }
 
     // Expand the range to fully include all contentEditable=False elements.
```

Comment:
the `expandSelectionToIncludeNonEditables` should probably move into the selection plugin at some point